csfle/java/aws/reader/src/main/java/com/mongodb/csfle/InsertEncryptedDocument.java

```diff
@@ -44,56 +44,54 @@
  * - Attempts to find the upserted document with the normal client using an encrypted field
  * - Finds the upserted document with the normal client using a non-encrypted field
  */
-public class insertEncryptedDocument {
+public class InsertEncryptedDocument {
```

Comment:
This PR renames Java files [as was done here](https://github.com/10gen/docs-in-use-encryption-generation/pull/17#discussion_r958614680)